src/manager/monitor.c

```diff
@@ -37,11 +37,6 @@ Subscription *create_monitor_subscription(Monitor *monitor, const char *node) {
         subscription->monitor = monitor_ref(monitor);
         subscription->free_monitor = (free_func_t) monitor_unref;
 
-        subscription->handle_unit_new = monitor_on_unit_new;
```

Comment:
Ideally you should break out this change to a separate commit.

Reply:
Done.